### ISSUE TITLE
No 120 fps optione

### ISSUE THREAD

**Issue by Trunk68** [KyroFrCode/moonlight-chrome-tizen-docker#11]
Hello guys, i have installed moonlight on my samsung tv (2021). Bu i can't find 120 fps in the options. Do you know the reason?

**Comment by KyroFrCode** [comment]
Did you reinstall the app with the current last update repository of moonlight-chrome-tizen ?

**Comment by Trunk68** [comment]
Yes, but there is no 120 fps option.. and if i try to add the 120 fps option manually like here (https://github.com/KyroFrCode/moonlight-chrome-tizen/pull/9), the Application just freezes at startup.

**Comment by SherifG97** [comment]
I too would love the 120hz option. I hoped this would be available as there are tizen TVs that are 120hz
